### PR TITLE
Fix signing due to subset projects rename

### PR DESCRIPTION
**Pull request by safern**
This fixes the official build by fixing signing. This was a fall out from: https://github.com/dotnet/runtime/commit/f53c1565

Test build is: https://dev.azure.com/dnceng/internal/_build/results?buildId=653965&view=results

cc: @dotnet/runtime-infrastructure

### CODE REVIEW

**Review by stephentoub — APPROVED** [review]

**Review by ViktorHofer — APPROVED** [review]

**Comment by stephentoub** [comment]
Thanks for fixing this. After merging this, can we kick off an official build?

**Comment by safern** [comment]
> Thanks for fixing this. After merging this, can we kick off an official build?

Yeah, I'll make sure to cancel whatever build is going on, and that will cause to automatically start a new one with this fix.

**Comment by hoyosjs** [comment]
@safern are these paths touched in our PR pipeline? Given that you created an internal validation build, does it make sense to just merge?

**Comment by safern** [comment]
> @safern are these paths touched in our PR pipeline? Given that you created an internal validation build, does it make sense to just merge?

They're not, but I want to wait for signing validation to finish just to be 100% sure it is fixed. Once that succeeds I'll merge.

**Comment by safern** [comment]
Signing validation is green, merging to unblock official builds:

image

**Pull request merged**
